dandiapi/api/views/dandiset.py

```diff
@@ -94,8 +95,21 @@ def create(self, request):
         if created:
             version_metadata.save()
 
-        dandiset = Dandiset()
-        dandiset.save()
+        if 'identifier' in serializer.validated_data['metadata']:
```

Comment:
I actually think that if we're going to support this feature first hand, we should specify it at the top level (next to `name`). What do you think?

Reply:
FWIW I think this feature is worthwhile to have only during migration although may be we could find some use later but it then must be restricted to admin only... As such I think may be it is not worth "promoting" it etc by placing to the top level.

Reply:
Requiring `identifier` as a top level field would involve tweaking the `VersionMetadataSerializer` and changing the CLI upload code, which doesn't seem like a worthwhile investment of time for a temporary tool like this.